docs/source/app_developers_guide/environment_setup.rst

```diff
@@ -0,0 +1,207 @@
+
+.. _environment_setup:
+
+*****************
+Environment Setup
+*****************
+
+Overview
+========
+
+This tutorial walks through the process of setting up Hyperledger Sawtooth for
+the purposes of app development. If you want to set up your environment for
+core development, please continue with the :doc:`/core_developers_guide`.
+
+At the end, you will be ready to perform app development tasks, such as
+creating new transaction families or handlers as described in the :ref
+:`transaction-family-tutorial`.
+
+Commands in this tutorial can be run via Terminal.app on MacOS, Git Bash on
+Windows, etc.
+
+
+Installing Sawtooth on Ubuntu Linux
+===================================
+
+You can install Sawtooth directly on your Ubuntu Linux machine, using the following steps:
+
+Prerequisites
+-------------
+
+Ubuntu 16.04 or later
+
+
+Installation
+------------
+
+Run the following commands from a terminal window:
+
+.. code-block:: console
+
+  $ echo "deb http://repo.sawtooth.me/ubuntu/0.8/stable xenial universe" >> /etc/apt/sources.list
+  $ apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 6B58B1AC10FB5F63
+  $ apt-get update && apt-get install -y sawtooth
+
+
+Building sawtooth-core
+----------------------
+
+Most of the components of Sawtooth Lake depend partially on code that must
+first be built. This includes C++/swig code and generating protobuf classes for
+each language. Sawtooth Lake also includes SDKs for other languages, including
+Java and JavaScript, which require building. To build the python portion of
+Sawtooth Lake within vagrant, do:
+
+.. code-block:: console
+
+  $ /project/sawtooth-core/bin/build_python
```

Comment:
If you installed sawtooth using apt-get, you do not have the `/project/sawtooth-core` directory or any of the build scripts. This step cannot be performed.